test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp/Models/DictionaryObjectModel.cs

```diff
@@ -10,5 +10,8 @@ public class DictionaryObjectModel
         public KeyValuePair<string, Int32ObjectModel> Int32ObjectValue { get; set; }
 
         public IDictionary<string, StringObjectModel> StringObjectModel { get; set; }
+
+        public Dictionary<int, int[]> IntArrayValue { get; set; }
```

Comment:
In the OpenAPI spec, the Dictionary type is always Dictionary<string, ....>. It can't be Dictionary<object, ...>.